www/src/components/banner.js

```diff
@@ -44,7 +44,7 @@ const Banner = () => (
   >
     <InnerContainer>
       <Content>
-        {`New! Create a site in 5 minutes with `}
+        {`New! Get 20x faster builds with `}
```

Comment:
Small note, unfortunately not entirely sure, but I think you could use the `×` https://en.wikipedia.org/wiki/Multiplication_sign instead of `x`? 🤔

Reply:
Ahhh no, just doesn't read as well — let's ignore semantics (that I'm not even 100% sure abt): 
![image](https://user-images.githubusercontent.com/21834/73566088-4ba86680-4463-11ea-8c6c-4ac5efa99a10.png)